src/WordPress.php

```diff
@@ -247,6 +247,53 @@ public function create( $args, $assoc_args ) {
 		\EE\Utils\delem_log( 'site create end' );
 	}
 
+	/**
+	 * Enable object cache.
+	 */
+	private function enable_object_cache() {
+		$redis_plugin_constant    = 'docker-compose exec --user=\'www-data\' php wp config set --type=variable redis_server "array(\'host\'=> \'ee-global-redis\',\'port\'=> 6379,)" --raw';
+		$activate_wp_redis_plugin = "docker-compose exec --user='www-data' php wp plugin install wp-redis --activate";
+		$enable_redis_cache       = "docker-compose exec --user='www-data' php wp redis enable";
+
+		$this->docker_compose_exec( $redis_plugin_constant, 'Unable to download or activate wp-redis plugin.' );
+		$this->docker_compose_exec( $activate_wp_redis_plugin, 'Unable to download or activate wp-redis plugin.' );
+		$this->docker_compose_exec( $enable_redis_cache, 'Unable to enable object cache' );
+	}
+
+	/**
+	 * Enable page cache.
+	 */
+	private function enable_page_cache() {
+		$activate_nginx_helper    = 'docker-compose exec --user=\'www-data\' php wp plugin install nginx-helper --activate';
+		$nginx_helper_fail_msg    = 'Unable to download or activate nginx-helper plugin';
+		$salt_value            = $this->site_data['site_url'] . ':';
+		$add_hostname_constant = "docker-compose exec --user='www-data' php wp config set RT_WP_NGINX_HELPER_REDIS_HOSTNAME ee-global-redis --add=true --type=constant";
+		$add_port_constant     = "docker-compose exec --user='www-data' php wp config set RT_WP_NGINX_HELPER_REDIS_PORT 6379 --add=true --type=constant";
+		$add_prefix_constant   = "docker-compose exec --user='www-data' php wp config set RT_WP_NGINX_HELPER_REDIS_PREFIX nginx-cache: --add=true --type=constant";
+		$add_cache_key_salt    = "docker-compose exec --user='www-data' php wp config set WP_CACHE_KEY_SALT $salt_value --add=true --type=constant";
```

Comment:
@mbtamuli `wp-cli` already sets it by default to random string. Just verify it once that we actually need it. It might be possible that we were doing it when wp was not automatically adding it. Since it adds everytime we generate config, we _might_ not need it

Reply:
We will add this to that clearing object cache per site in case of global redis.